sources/index.html

```diff
@@ -1,230 +1,315 @@
+            <ul class="container-projects-skill">
+              <li class="secs">Bootstrap</li>
+              <li class="secs">Ruby on rails</li>
+              <li class="secs">Rspec</li>
+            </ul>
+          </div>
+          <div class="container-projects">
+            <ul class="container-projects-skill">
+              <li class="secs">Capybara</li>
+              <li class="secs">Selenium</li>
+            </ul>
+          </div>
+        </section>
+
+        <section class="about-container">
+          <img src="../images/icon-skills.svg" alt="icon" />
+          <div class="container-title">
+            <h4>Skills</h4>
+          </div>
+          <div class="container-projects">
+            <ul class="container-projects-skill">
+              <li class="secs">Codekit</li>
+              <li class="secs">Github</li>
+              <li class="secs">Codepen</li>
+              <li class="secs">Gitlab</li>
+            </ul>
+          </div>
+          <div class="container-projects">
+            <ul class="container-projects-skill">
+              <li class="secs">Terminal</li>
+            </ul>
+          </div>
+        </section>
+      </section>
+    </section>
+    <!--start contact section-->
+    <footer class="contact-container">
+      <h3>
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h3>
+      <form action="https://formspree.io/f/xeqnejed" method="post">
+        <ul class="form-input">
+          <li>
+            <label for="name"></label>
+            <input
+              class="input"
+              type="text"
+              name="user-name"
+              id="name"
+              placeholder="Full name"
+              required
+              maxlength="30"
+            />
+          </li>
+          <li>
+            <label for="email"></label>
+            <input
+              class="input"
+              type="email"
+              name="user-email"
+              id="email"
+              placeholder="Email address"
+              required
+            />
+          </li>
+          <li>
+            <label for="client-message"></label>
+            <textarea class="text-area"
+              name="message"
+              id="client-message"
+              cols="30"
+              rows="10"
+              placeholder="Enter text here"
+              required
+              maxlength="500"
+            ></textarea>
+          </li>
+
+          <li>
+            <button type="submit" class="submit-message-button">
+              Get in touch
+            </button>
+          </li>
+        </ul>
+
+        <ul class="social-icons">
+          <li><i class="fab fa-twitter"></i></li>
+          <li><i class="fab fa-linkedin"></i></li>
+          <li><i class="fab fa-medium"></i></li>
+          <li><i class="fab fa-github"></i></li>
+          <li><i class="fab fa-angellist"></i></li>
+        </ul>
```

Comment:
- [ ] [OPTIONAL] Please fix the horizontal scroll bar on your design so that it can look like the provided [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468)
- [ ] [OPTIONAL] Kindly fix the padding in between these social media icons as they seem to be different from the padding provided in the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A468). See below:
### Your design
![your-design](https://user-images.githubusercontent.com/58692590/149037713-bae10ce6-e7ec-46e9-ab55-36bf4efea92f.png)

### Figma Template
![figma1](https://user-images.githubusercontent.com/58692590/149037729-b207937a-4666-48f2-b9ba-035e7ea4bcdf.png)